frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -439,18 +439,17 @@ Object.assign(SmartScalar, {
   },
 
   // Smart scalars need to have a breakout
-  checkRenderable(
-    [
-      {
-        data: { insights },
-      },
-    ],
-    settings,
-  ) {
+  checkRenderable([
+    {
+      data: { insights },
+    },
+  ]) {
     if (!insights || insights.length === 0) {
-      throw new NoBreakoutError(
+      throw new ChartSettingsError(
```

Comment:
If we want to display the empty state instead of an error, the error thrown from `checkRenderable` must be the instance of `ChartSettingsError`!